code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -341,11 +341,13 @@
 		"<span class='warning'>You [actuallyismob ? "try to ":""]stuff [O] into [src].</span>", \
 		"<span class='hear'>You hear clanging.</span>")
 	if(actuallyismob)
+		var/mob/living/L = O
+		if(L.mob_size > max_mob_size)
+			return
```

Comment:
Provide a message that says why it didn't work.

Reply:
![image](https://user-images.githubusercontent.com/61567407/189991543-ba261bd0-c0c0-4d91-83fd-b588f2d531c1.png)